### ISSUE TITLE
App loses functionality when Initate Media sync on app launch is enabled 

### ISSUE THREAD

**Issue by ZeroShadowX** [sircharlo/jw-meeting-media-fetcher#208]
### Describe the bug
When **Initate Media sync on app launch** is enabled the app in Settings the App will lose functionality 

When this option is enabled you will not able to download any media. I noticed that the app will seem to constantly ping jw.org but nothing will happen. As you can see in the screenshots the jw.org area will be orange and will constanly spin but never change, I tried different internet connection (wifi/Lan) but no change. 

 I also noticed that the Home icon in the bottom right will be gone but you can still click on it. Also observed that you are unable to select any week at the bottom of the app as well. 

Can be fixed by not enabling that option. 

### To Reproduce
1. Open App
2. Go to Settings enable Initiate media Sync on app Launch 
3. Close App 
3. Open App
4. See behavior 

### Screenshots
[(https://drive.google.com/file/d/1Gifqj3XVM9x276D3mxImW8X3IhLvh_69/view?usp=sharing)]

[https://drive.google.com/file/d/1TGlX8L0nDdV5a5cYz4GZmsO9mQUnQBfu/view?usp=sharing]

### System specs
- Windows_NT 10.0.19043
- JWMMF v22.2.0

### Additional context
Once you disable that option the program will function as normal. 

### Anonymized `prefs.json`
```
{
  "autoOpenFolderWhenDone": true,
  "autoQuitWhenDone": null,
  "autoRunAtBoot": null,
  "autoStartSync": null,
  "congServer": null,
  "congServerPass": null,
  "congServerPort": null,
  "congServerUser": null,
  "enableMp4Conversion": true,
  "enableMusicButton": null,
  "enableMusicFadeOut": null,
  "excludeLffi": true,
  "excludeLffiImages": true,
  "excludeTh": true,
  "keepOriginalsAfterConversion": null,
  "lang": "E",
  "langUpdatedLast": "2022-02-20T17:11:00.112Z",
  "localAdditionalMediaPrompt": null,
  "localAppLang": "en",
  "localOutputPath": "***",
  "maxRes": "720p",
  "musicFadeOutTime": null,
  "musicFadeOutType": null,
  "mwDay": "3",
  "mwStartTime": "07:30",
  "weDay": "5",
  "weStartTime": "10:30"
}
```

**Comment by sircharlo** [comment]
Hi there! Thanks for reporting this. 

I had noticed this issue; it's already been fixed [here](https://github.com/sircharlo/jw-meeting-media-fetcher/commit/dc9b8040f0b6d0165078cce96f71fe46564fb590#diff-d79af92b5cabad1e91c6c81c560d2151b37e3b2810d5a384e46964dc0cf51761L1193) and the fix will be released with the next update. 
This bug is relatively limited in scope. It only happens if attempting a media sync for any given week *after* all meeting days for that week. Example: meeting days are Thursday and Saturday, attempt a media sync on Sunday for that current week. The bug presents itself when "auto sync on launch" is enabled because auto-sync attempts to sync th current week by default.

Long story short, it will be resolved soon, but shouldn't affect real-world usage too much in the meantime 😃
